packages/docs/src/pages/startup/sass-and-css.md

```diff
@@ -8,13 +8,13 @@ weight: 1
 The easiest way to add the design system's styles to your site is by referencing its minified CSS.
 
 1. Download the zip file from the latest CMS design system release and open that file.
-1. Copy the `packages/core/dist/index.css` file into a relevant place in your code base — likely a directory where you keep third-party libraries. In the example below, our directory is `css/vendor`.
+1. Copy the `packages/cmsds/dist/index.css` file into a relevant place in your code base — likely a directory where you keep third-party libraries. In the example below, our directory is `css/vendor`.
```

Comment:
Should we rename the package folder again to just `design-system`? At some point I'm thinking this could be confusing.

Reply:
hm thats a good point, i like that its consistent with the package name. I'd like to get @line47 @davidakennedy thoughts too

Reply:
ok, now that we clarified some things in the CMS slack, even if we distribute the `dist` folder as originally planned, doesnt that mean the user doesnt have to navigate through the `packages` folder? The zip on the website probably shouldnt have that right?

Reply:
Yeah, I think it is indeed a good idea to pare that zip on github down to something like
```
zip.zip
  ├── design-assets
  ├── dist
      ├── index.js
      ├── index.scss
      ├── etc...
  ├── examples
```
We don't have to call it `dist` here, but I can't think of another name

Reply:
Or even just the contents of the `dist` folder inside the root of the zip, but I don't know if we would want to leave the other stuff out because of designers.